### ISSUE TITLE
[Woo POS] Simple products modal is unresponsive after coming back from store management

### ISSUE THREAD

**Issue by staskus** [woocommerce/woocommerce-ios#14528]
**Describe the bug**

Simple products modal is unresponsive after coming back from store management

**To Reproduce**
1. Open POS
2. Tap information (i) icon on the top of the product list
3. Confirm a modal appears
4. Tap "Create an order in store management"
5. Come back from store management from POS
6. Notice OK button is not responsive

There's a workaround, a modal can still be dismissed by tapping on the background

**Screenshots**

https://github.com/user-attachments/assets/fa75833d-23fe-414a-9014-b03b78dd4bf9

**Expected behavior**

Modal actions should be responsive after coming back from the store management.

**Mobile Environment**
Please include:
- Device: iPad Air M2
- iOS version: iOS 18.1
- WooCommerce iOS version: 21.2

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by joshheald** [comment]
No longer reproducible now that we use `fullscreenCover` for POS and dismiss it when we move to order creation.

@staskus could you double check with iOS 18 if you have a mo? My iPad still has 17

**Comment by staskus** [comment]
> No longer reproducible now that we use fullscreenCover for POS and dismiss it when we move to order creation.

@joshheald Yes, the POS is closed completely now if we leave it to other tabs. I am not sure if that's a perfect behavior but we can leave any decision for flows between POS and other parts of application for later.
